homeassistant/components/netatmo/sensor.py

```diff
@@ -490,6 +510,62 @@ def async_update_callback(self) -> None:
         self._attr_native_value = self._module.battery
 
 
+class NetatmoSensor(NetatmoBase, SensorEntity):
+    """Implementation of a Netatmo sensor."""
+
+    def __init__(
+        self,
+        netatmo_device: NetatmoDevice,
+        description: NetatmoSensorEntityDescription,
+    ) -> None:
+        """Initialize the sensor."""
+        super().__init__(netatmo_device.data_handler)
+        self.entity_description = description
+
+        self._module = netatmo_device.device
+        self._id = self._module.entity_id
+
+        self._publishers.extend(
+            [
+                {
+                    "name": HOME,
+                    "home_id": netatmo_device.device.home.entity_id,
+                    SIGNAL_NAME: netatmo_device.signal_name,
+                },
+            ]
+        )
+
+        self._attr_name = f"{self._module.name} {self.entity_description.name}"
+        self._room_id = self._module.room_id
+        self._model = getattr(self._module.device_type, "value")
+        self._config_url = CONF_URL_ENERGY
+
+        self._attr_unique_id = (
+            f"{self._id}-{self._module.entity_id}-{self.entity_description.key}"
+        )
+
+    @callback
+    def async_update_callback(self) -> None:
+        """Update the entity's state."""
+        try:
+            if (state := getattr(self._module, self.entity_description.key)) is None:
```

Comment:
Change this to hasattr?

Reply:
Yeah, for that line alone, yes. But I use `state` two lines later.